static/css/business.less

```diff
@@ -0,0 +1,38 @@
+/* ==========================================================================
+   cfgov-refresh
+   Business with us pages at /doing-business-with-us/.
+   ========================================================================== */
+
+// Used to set padding to 30px
+@padding: unit(@grid_gutter-width / @base-font-size-px, em);
```

Comment:
What do you propose? That's already the point of `@grid_gutter-width`, and you have to do the calculation every time, if you want ems, because you can't be sure what the base font size is.